File: app/src/main/java/com/neome/api/meta/base/dto/FieldValueVideo.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MediaIdImage;
import com.neome.api.meta.base.Types.MediaIdVideo;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class FieldValueVideo
{
  public long durationMs;

  @Nullable
  public String fileName;

  public MediaIdImage mediaIdBlurImage;

  public MediaIdImage mediaIdImage;

  public MediaIdVideo mediaIdVideo;

  public String primaryColor;

  public long size;
}

File: app/src/main/java/com/neome/api/meta/base/dto/FieldValueVideo.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MediaIdImage
import com.neome.api.meta.base.Types.MediaIdVideo

interface FieldValueVideo
{
  val durationMs: Long
  val fileName: String?
  val mediaIdBlurImage: MediaIdImage
  val mediaIdImage: MediaIdImage
  val mediaIdVideo: MediaIdVideo
  val primaryColor: String
  val size: Long
}